packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -204,6 +208,12 @@ class _SnapshotViewState extends State<SnapshotView> with AutoDisposeMixin {
     });
 
     addAutoDisposeListener(controller.snapshotJsonFile);
+
+    addAutoDisposeListener(controller.processingNotifier, () {
+      setState(() {
+        processing = controller.processingNotifier.value;
+      });
+    });
```

Comment:
instead of setting state for the whole widget, why not just use a ValueListenableBuilder widget that handles the state management for processing?

same comment below

Reply:
Makes sense! done